polaris-service/src/main/java/org/apache/polaris/service/BootstrapRealmsCommand.java

```diff
@@ -48,17 +46,11 @@ protected void run(
       Bootstrap<PolarisApplicationConfig> bootstrap,
       Namespace namespace,
       PolarisApplicationConfig configuration) {
-    MetaStoreManagerFactory metaStoreManagerFactory = configuration.getMetaStoreManagerFactory();
+    MetaStoreManagerFactory metaStoreManagerFactory =
```

Comment:
Why not `@Inject` these?

Reply:
Dropwizard Commands aren't managed by HK2. We instantiate them directly in `PolarisApplication`.

Reply:
But that means that there's a third (CDI, Jackson, manual) way to get those instances. I think, CDI should be used everywhere (except for very specific tests).

Reply:
Commands (boostrap/purge) are going to be fun to adapt to CDI. I suggest we look into them later but my feeling is that we would need to move them to separate modules.